docs/bucket/notifications/README.md

```diff
@@ -372,36 +372,72 @@ go run nats.go
 <a name="PostgreSQL"></a>
 ## Publish Minio events via PostgreSQL
 
-Install PostgreSQL from [here](https://www.postgresql.org/).
+Install [PostgreSQL](https://www.postgresql.org/) database server. For illustrative purposes, we have set the "postgres" user password as `password` and created a database called `minio_events` to store the events.
+
+This notification target supports two formats: _namespace_ and _access_.
+
+When the _namespace_ format is used, Minio synchronizes objects in the bucket with rows in the table. It creates rows with two columns: key and value. The key is the bucket and object name of an object that exists in Minio. The value is JSON encoded event data about the operation that created/replaced the object in Minio. When objects are updated or deleted, the corresponding row from this table is updated or deleted respectively.
+
+When the _access_ format is used, Minio appends events to a table. It creates rows with two columns: event_time and event_data. The event_time is the time at which the event occurred in the Minio server. The event_data is the JSON encoded event data about the operation on an object. No rows are deleted or modified in this format.
+
+The steps below show how to use this notification target in `namespace` format. The other format is very similar and is omitted for brevity.
 
 ### Step 1: Add PostgreSQL endpoint to Minio
 
-The default location of Minio server configuration file is ``~/.minio/config.json``. Update the PostgreSQL configuration block in ``config.json`` as follows:
+The default location of Minio server configuration file is ``~/.minio/config.json``. The PostgreSQL configuration is located in the `postgresql` key under the `notify` top-level key. Create a configuration key-value pair here for your PostgreSQL instance. The key is a name for your PostgreSQL endpoint, and the value is a collection of key-value parameters described in the table below.
+
+| Parameter | Value type | Description |
+|:---|:---|:---|
+| `enable` | Boolean | (Required) Is this server endpoint configuration active/enabled? |
```

Comment:
This value can be represented with Go's native type. as '_bool_'  and '_string_' .  Also you can just call 'Value type' as just 'Type'